superbench/common/devices/__init__.py

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+"""SuperBench devices module."""
+
+from superbench.common.devices.gpu import GPU
+
+__all__ = [
+    'GPU',
+]
```

Comment:
Why do we put \_\_init\_\_.py in common directly. Currently, we reduce it from `superbench.common.devices.gpu.GPU` to `superbench.common.devices.GPU`

Reply:
Besides, why not put into `utils`?